scripts/install

```diff
@@ -26,6 +26,13 @@ ynh_script_progression --message="Installing dependencies... This might take som
 
 ynh_install_nodejs --nodejs_version=$NODEJS_VERSION
 
+#=================================================
+# CREATE A POSTGRESQL DATABASE
+#=================================================
+ynh_script_progression --message="Initializing postgresql modules..."
+
+ynh_psql_execute_as_root --sql="CREATE EXTENSION IF NOT EXISTS pgroonga;" --database=$db_name
```

Comment:
I just discovered that PGroonga is not an additional module or extension that comes with PostgreSQL. I am not sure whether this extension is installed upstream.
If the upstream is not installed, you need to refer to https://pgroonga.github.io/install/ to supplement the PGroonga installation part.